pliers/extractors/api.py

```diff
@@ -31,6 +36,7 @@ class IndicoAPIExtractor(Extractor):
 
     _log_attributes = ('models',)
     _input_type = ()
+    _batch_size = 20
```

Comment:
Is there an easy way to make this an initialization argument for any class that implements `BatchTransformerMixin`? Can we check `kwargs` for a `batch_size` argument, and overwrite the class-level `_batch_size` value if it's found?